landscape/client/manager/livepatch.py

```diff
@@ -22,10 +21,30 @@ class LivePatch(DataWatcherManager):
     run_interval = 1800  # Every 30 min
 
     def get_data(self):
-        json_output = get_livepatch_status('json')
-        readable_output = get_livepatch_status('humane')
-        return json.dumps({'humane': readable_output, 'json': json_output},
-                          sort_keys=True)  # Prevent randomness for cache
+        json_output = get_livepatch_status("json")
+        readable_output = get_livepatch_status("humane")
+        return json.dumps(
+            {"humane": readable_output, "json": json_output}, sort_keys=True
+        )  # Prevent randomness for cache
+
+
+def _parse_humane(output):
+    entries = output.split("\n")
+    data = {}
+
+    for e in entries:
+        line = e.strip()
+        if len(line) == 0:
```

Comment:
len(line) == 0 would always evaluate to empty string '' so it's more clear to `if not line` or to `if line == ''`